src/lib/header-bidding/prebid/prebid.spec.ts

```diff
@@ -32,7 +32,6 @@ const resetPrebid = () => {
 describe('initialise', () => {
 	beforeEach(() => {
 		resetPrebid();
-		window.guardian.config.switches.commercial = true;
```

Comment:
We don't need to replace with `switches.shouldLoadGoogletag` here?

Reply:
It's not actually required in this test, so I removed it entirely. I think it's only necessary in prepare-prebid.spec.ts